Dockerfile

```diff
@@ -1,20 +1,28 @@
-FROM node:16
+# Build Stage
+FROM node:16 AS build
 
-# Create app directory
 WORKDIR /usr/src/app
 
-# Install app dependencies
-# A wildcard is used to ensure both package.json AND package-lock.json are copied
-# where available (npm@5+)
 COPY package*.json ./
 
 RUN npm ci
 
-# Bundle app source
 COPY . .
 
 RUN npm run build
 
+# Run Stage
+FROM node:16-alpine
+
+WORKDIR /usr/src/app
+
+COPY .env* .
```

Comment:
This line is broke yo

I dunno how to make this work both locally & in production because I don't know how to do environment variables in docker

Reply:
I ended up just going with the hack way anyway. Now its merged into one copy command: `COPY package*.json .env* ./` which works since it only requires one of the sources to have contents for the command to be successful